api/v1/checlusterbackup_types.go

```diff
@@ -42,6 +42,9 @@ type CheClusterBackupStatus struct {
 	// Last backup snapshot ID
 	// +optional
 	SnapshotId string `json:"snapshotId,omitempty"`
+	// Version that was backed up
+	// +optional
+	CheVersion string `json:"cheVersion,omitempty"`
```

Comment:
I don't find any evidence why we need this.

Reply:
This information added in the status. Needed for chectl to not to download and decrypt backup snapshot.